File: BlockChain.Domain/BlockChain/Nft.cs
using BlockChain.Cross.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BlockChain.Domain.BlockChain
{
    public class Nft : Entity<Guid>
    {
        public string Nome { get; set; }
        public string Imagem { get; set; }
        public string Rank { get; set; }
    }
}

File: frontend/src/app/model/Nft.ts
import { BaseEntity } from './BaseEntity';

/*import { TipocomandOperacao } from './TipocomandOperacao';
import { TipoMotivorestricao } from './TipoMotivorestricao';
import { ComandOperacao } from './ComandOperacao';*/


export class Nft extends BaseEntity<number> {


    public Nome?: string;

    public Imagem?: string;
    
    public Rank?: number;

    public IdRede?: number;  
    

}